Rules.md

```diff
@@ -5344,6 +5345,63 @@ func foo() -> [String] {
 
 
 
+## Explicit Self
+
+Identifier | Enabled by default | Supports autocorrection | Kind | Minimum Swift Compiler Version
+--- | --- | --- | --- | ---
+`explicit_self` | Disabled | Yes | style | 3.0.0 
```

Comment:
we probably should list if this is an `analyzer` rule here (but we can do it later in another PR)

Reply:
Done